src/index.js

```diff
@@ -1,6 +1,17 @@
-const fibonacci = (n) => {
+const fibonacci = n => {
+  if (n === 0 || n === 1) {
+    return [1]
+  }
+
+  const sequence = [1, 1]
+  let next = 2
 
+  while (sequence.length !== n) {
```

Comment:
Este while solo funciona cuando n es un número, si pones otro tipo de dato entra en un ciclo infinito. Te cuidad con el while. En este caso tienes bien determinado el número de pasos, por lo que te recomiendo mejor un for.

Reply:
No pensé en ese caso. Gracias por el feedback!